packages/devtools_app/lib/main.dart

```diff
@@ -43,6 +44,11 @@ void main() async {
   setGlobal(DevToolsExtensionPoints, ExternalDevToolsExtensionPoints());
   setGlobal(IdeTheme, getIdeTheme());
 
+  // inspectorPreferences depends on the IdeTheme global
+  // to initialize
```

Comment:
Make the comment complete sentence: start with capital letter and finish with '.'